tsconfig.publish.json

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "./tsconfig.json",
```

Comment:
TIL.

Reply:
Yeah it means my testing builds don't get plagued with things like unused locals etc when I'm just trying to develop. It's pretty nice :)